### ISSUE TITLE
YoloV8 Classification learning rate

### ISSUE THREAD

**Issue by djaygier** [ultralytics/ultralytics#5007]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How do i set the learning rate?

When using this code for example

`from ultralytics import YOLO, checks, hub

checks()

hub.login('') // my login key

model = YOLO('https://hub.ultralytics.com/models/')

model.train()`

How do i set the lr0 and lrf? doing

`model.train(lr0=0.001, lrf=0.001)`

Doesn't seem to work.

I need a lower learning rate for my model.

**Comment by RizwanMunawar** [comment]
Hello @djaygier, I'd like to clarify that if you're training the model using the Ultralytics Hub's `Bring your own agent` option, please note that customizing arguments is currently not supported. Thank you for bringing this to our attention, and rest assured that our team is actively working on resolving this issue, and we will provide a solution in the near future.

You have the option to address this problem using an alternative approach, and the sample code is provided for your reference.

```python
from ultralytics import YOLO

# Load a model
model = YOLO('yolov8n-cls.yaml')  # build a new model from YAML
model = YOLO('yolov8n-cls.pt')  # load a pretrained model

# Train the model
  results = model.train(data='mnist160', epochs=100, imgsz=64, lr0=0.00777,
                        lrf=0.001)
```

And here's the relevant part of the output that shows the training parameters:

**When Training Started**
```
task=classify, mode=train, model=yolov8n-cls.pt, data=mnist160, epochs=100, patience=50, batch=16, imgsz=64,\
nms=False, lr0=0.00777, lrf=0.001, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, \
warmup_momentum=0.8, warmup_bias_lr=0.1, save_dir=runs\classify\train2
```
As you can see, the `lr0` and `lrf` values have indeed changed as per the provided values during training.

Thanks
